raddb/mods-available/magic

```diff
@@ -0,0 +1,12 @@
+# When the Login-Time attribute is set to some value, and the
+# user has been permitted to log in, a Session-Timeout is
+# calculated based on the remaining time.  See "doc/README".
+#
+magic {
+    is_local = true
+    # The ip or url of the magic gateway
+    # socket-address = 127.0.0.1
```

Comment:
everything else uses `server` for the configuration item which contains the name of the external server.  If you make your module "unusual" to configure, then people will be confused by it.

You should just use `server`, and drop the `is_local` setting.  If the `server` starts with a `/`, then it's a local Unix socket.  Otherwise it's not.